mirai-core-api/src/commonMain/kotlin/contact/Group.kt

```diff
@@ -175,6 +176,45 @@ public interface Group : Contact, CoroutineScope, FileSupported {
      */
     public suspend fun setEssenceMessage(source: MessageSource): Boolean
 
+    /**
+     * 获取所有群公告列表
+     */
+    @MiraiExperimentalApi
+    public suspend fun getGroupAnnouncements(): List<Announcement>
+
+    /**
+     * 发送群公告
+     *
+     * @param announcement [Announcement] 公告的信息设置
+     *
+     * @throws PermissionDeniedException 没有权限时抛出
+     *
+     * @return 公告的fid
+     */
+    @MiraiExperimentalApi
+    public suspend fun sendGroupAnnouncement(announcement: Announcement): String
```

Comment:
`Group` 对象中的方法就可以不包含 `Group` 命名了
```suggestion
    public suspend fun sendAnnouncement(announcement: Announcement): String
```